prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,30 @@
+from kivy.app import App
+from kivy.app import Builder
+from kivy.uix.label import Label
+from random import randint
+
+
+class DynamicLabels(App):
+
+    def __init__(self):
+        super().__init__()
+        self.names = ['Caleb', 'Alebc', 'Lebca', 'Ebcal', 'Bcale']
+
```

Comment:
__init__ method seems correct, it calls the superclass and creates the list of names.